src/ProcessManagerWindow.js

```diff
@@ -48,6 +48,15 @@ class ProcessManagerWindow extends BrowserWindow {
 
       this.emit('kill-process', pid);
     });
+    ipcMain.on('process-manager:debug-process', (e, pid) => {
+      // ignore if not for us
+      if (!this || this.isDestroyed()) return;
+      if (e.sender !== this.webContents) return;
+
+
+      this.emit('debug-process', pid);
```

Comment:
Suggestion: I'd be more confortable passing the `webContentsId` all over rather than the `pid`